docs/source/conf.py

```diff
@@ -899,8 +899,8 @@ def _resolve_arg(self, arg: Any) -> Union[type, None]:
             dict,
             tuple,
             Union,
-            collections.Coroutine,
-            collections.Sequence,
+            collections.abc.Coroutine,
+            collections.abc.Sequence,
             typing.Sequence,
```

Comment:
let's use collections.abc.Sequence here as well since this is deprecated in py 3.9 (which is what the docs build out of)

Reply:
Sorry, I didn't get that :)

I now just `import collections.abc` and then use things like `collections.abc.Coroutine` in code.

Reply:
my bad, I meant that there is `typing.Sequence` and `collections.abc.Sequence` in that list. Do we need both of them?

Reply:
Well in practice probably not because we don't have any args that are `typing.Sequence`. But if there were any, then `typing.Sequence` and `collections.abc.Sequence` wouldn't be the same.